### PR TITLE
add 3.8.14

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

As per vote in https://github.com/conda-forge/python-feedstock/pull/579

### CODE REVIEW

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

**Comment by jaimergp** [comment]
@conda-forge/python - this is scheduled for release on Nov 11th or later, but if there are any items to review here, we can start now. Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "3.8.13" %}
+{% set version = "3.8.14" %}
```
Comment (jakirkham) [inline recipe/meta.yaml:1]: Should we just go to 3.8.15?

Comment (mbargull) [reply recipe/meta.yaml:1]: Since the PRs (this one, 3.9.14, 3.10.7) are ready, I'd say let's just build them and update afterwards.

**Comment by mbargull** [comment]
@conda-forge-admin, please rerender

**Comment by raydouglass** [comment]
Hi @jaimergp @mbargull. Just wondering what the status of this PR is? Is it ready to be merged?

**Comment by mbargull** [comment]
@raydouglass, I'll merge this today after another re-rendering to include incorporate some other recent build system changes.

**Comment by mbargull** [comment]
@conda-forge-admin, please rerender

**Pull request merged**